deepnote_toolkit/runtime_patches.py

```
@@ -1,10 +1,64 @@
-from typing import Any, Optional, Union
+from typing import Any, Dict, List, Optional, Union
 
 from deepnote_toolkit.logging import LoggerManager
 
 logger = LoggerManager().get_logger()
 
 
+def _monkeypatch_trino_cancel_on_error():
```

Ruff (0.14.14)

[warning] Missing return type annotation for private function `_monkeypatch_trino_cancel_on_error`

Add return type annotation: `None`

(ANN202)